infra/build_specified_commit.py

```diff
@@ -61,3 +62,54 @@ def build_fuzzer_from_commit(project_name,
       env_to_add=None,
       source_path=old_repo_manager.repo_dir,
       mount_location=os.path.join('/src', old_repo_manager.repo_name))
+
+
+def detect_main_repo_from_docker(project_name, example_commit, src_dir='/src'):
+  """Checks a docker image for the main repo of an OSS-Fuzz project.
+
+  Args:
+    project_name: The name of the OSS-Fuzz project
+    example_commit: An associated commit SHA
+    src_dir: The location of the projects source on the docker image
+
+  Returns:
+    The repo's origin, the repo's name
+  """
+  helper.build_image_impl(project_name, no_cache=True)
+  docker_image_name = 'gcr.io/oss-fuzz/' + project_name
+  command_to_run = [
+      'docker', 'run', '--rm', '-i', '-t', docker_image_name, 'python3',
+      os.path.join(src_dir, 'detect_repo.py'), '--src_dir', src_dir,
+      '--example_commit', example_commit
+  ]
+  out, _ = run_command(command_to_run, check_result=True)
+  repo_info = out.split('\n')[-2]
+  pair = repo_info.split(' ')
+  # Commit did not exist in directory
+  if len(pair) != 2:
+    return None, None
+  return pair[0].rstrip(), pair[1].rstrip()
+
+
+def run_command(command, location='.', check_result=False):
```

Comment:
`.` is not good. Better use `location=None` and then in the function code do:

```
if not location:
  location = ...
```

either `os.getcwd()` or the path based on the `__file__`, depending on what default value you need here

Reply:
Done.